submissions/yhorodechna/side-menu-dom/js/script.js

```diff
@@ -0,0 +1,107 @@
+let _navEl;
+const getNavElement = () => {
+    if (!_navEl) {
+        _navEl = document.querySelector('#nav');
+    };
+    return _navEl;
+};
+let ACTIVE_EL_ID;
+
+function findHeaderLinkOrNavLink(clickedElement) {
+    let res;
+    let current = clickedElement;
+    while (!res && current) {
+        const { className } = current;
+        if (className) {
+            if (className.includes('nav__link')) {
+                res = {
+                    navLink: current
+                }
+                continue;
+            }
+            else if (className.includes('header__link')) {
+                res = {
+                    headerLink: current
+                }
+                continue;
+            }
+        }
+        current = current.parentNode;
+    }
+    return res;
+};
+
+function handleContainerClick({ target }) {
+    const { navLink, headerLink } = findHeaderLinkOrNavLink(target);
+
+    if (navLink) {
```

Comment:
you can join if statements in a single `if`

Reply:
Join.
Also I found Element.closest() method.  I used it instead while loop.